include/reloader.h

```diff
@@ -41,9 +41,6 @@ class Reloader {
 	/// made through \a easyhandle, unless it's nullptr, in which case
 	/// method creates a temporary handle that is destroyed when method
 	/// completes.
-	// TODO: check that the value passed via "max" is always obtained from
```

Comment:
Do we have an issue for this? I couldn't find it.

Reply:
The TODO seemed to point to a possible improvement (not a bug).
I implemented it in https://github.com/newsboat/newsboat/pull/1310/commits/84421408b37ac8c96bcb14e836332b9e38cd7b25.